jetpack.php

```diff
@@ -49,6 +49,53 @@
 defined( 'JETPACK_SIGNATURE__HTTP_PORT' )  || define( 'JETPACK_SIGNATURE__HTTP_PORT', 80 );
 defined( 'JETPACK_SIGNATURE__HTTPS_PORT' ) || define( 'JETPACK_SIGNATURE__HTTPS_PORT', 443 );
 
+// TODO
+//                          ,--,                     ,----,
```

Comment:
I suggest we remove this comment art, but I've opened #13435 to make sure we don't forget to revert this one.